### ISSUE TITLE
Docker Container not Stopping When Downloading OSM Planet Completed

### ISSUE THREAD

**Issue by YvesZumbach** [openmaptiles/openmaptiles-tools#332]
Hello openmaptiles,

I am using your docker container to download OSM planet. I was expecting that the container would stop after completing the download. However it did not and kept running.

I need the container to stop so that I can execute some more commands in the script I am writing. I was wondering if it is a mistake from my side that the container is still running or maybe if you could indicate to me how to make it stop after completing the download?

In any case, thanks a lot for this awesome set of tools!

**Comment by TomPohys** [comment]
Hi @YvesZumbach, could you please provide some more details? E.g how you start the container?

Thanks

**Comment by YvesZumbach** [comment]
Sure, I run `docker run --rm -v /mnt/disks/disk_name:/data openmaptiles/openmaptiles-tools download-osm planet -- -d /data`

**Comment by TomPohys** [comment]
Do you test it on some smaller area? I just run `docker run --rm -v /mnt/disks/disk_name:/data openmaptiles/openmaptiles-tools download-osm monaco -- -d /data` and the docker was stopped correctly. Or can you try something bigger like `switzerland` to have some time to check `docker ps`. 

Maybe some lack of privileges to stop a container?

**Comment by YvesZumbach** [comment]
Hi @TomPohys,

I tried downloading some data on my computer and, of course, you were absolutely right: the container stopped right after completing the download.

I think something went wrong when I tried downloading planet as I still had a `.aria` file hanging around which is normally only present during the download, but removed once completed if I understand things correctly. I also did not see the completed message in the logs that I saw when trying to download Monaco or Switzerland. I will therefore assume that something went wrong at the very end of the download process and try it again in the near future to assert that it was a one-time issue (maybe I should have waited more, maybe some server had an issue when I was downloading).

In any case, many thanks for your very quick and targeted help!
